src/Filter/Filter.php

```diff
@@ -2,12 +2,9 @@
 
 namespace Novaway\ElasticsearchClient\Filter;
 
-interface Filter
+use Novaway\ElasticsearchClient\Clause;
+
+interface Filter extends Clause
```

Comment:
Does this interface still revelent as there is not method here ?

Reply:
It is, at least for backward compatibility (you can create you own filters with this interface). 

If this PR is a major update, there is no need to keep it, if not, a `@deprecated` annotation might be useful.